terraform/modules/cloudfunction/src/requirements-dev.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pytest
```

Comment:
switch to poetry for consistency? probably requires:
- update instructions
- update makefile to run `poetry export` before terraform apply

Reply:
Since cloud functions only work natively with `requirements.txt`, I would suggest either:

1. Leave as-is with `requirements.txt` for simplicity
2. Change from Cloud Function -> Cloud Build (probably overkill given that we intend to remove this later when Vertex Scheduler is available in Terraform)